src/models/team_user.rs

```diff
@@ -3,18 +3,11 @@ use serde::Serialize;
 use super::{team::Team, user::User};
 use crate::schema::team_users;
 
-#[derive(Debug, Queryable, Serialize, Identifiable, Associations)]
+#[derive(Debug, Queryable, Serialize, Identifiable, Associations, Insertable)]
 #[belongs_to(Team)]
 #[belongs_to(User)]
 #[primary_key(team_id, user_id)]
 pub struct TeamUser {
     pub user_id: i32,
     pub team_id: i32,
 }
-
-#[derive(Debug, Insertable)]
-#[table_name = "team_users"]
```

Comment:
this should be moved to the above struct

Reply:
Diesel actually infers the table name from the struct name in this case (if there was an issue, it would be caught by the compiler)